agents/portfolio_intelligence.py

```
@@ -34,7 +34,27 @@
+    repo's old name, before the ``autonomous-ai-agency`` rename). Every
+    GitHub API call made with the stale name 404s, and
+    ``fetch_github_signals`` doesn't check the response status before
+    treating the JSON body as a list, so the 404 error object (a dict)
+    crashed with ``TypeError: unhashable type: 'slice'`` — silently
+    dropping all GitHub-sourced portfolio initiatives (bugs, PRs) on
+    every refresh.
+    """
+    try:
+        from packages.config import settings
+        return settings.github_repository
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)